From cc7e76dad36b8bea02bc0c56799b1eae1e8ccb09 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 18 Feb 2024 17:38:00 +0100
Subject: arm64: dts: sun50i-a64-pinetab: Name sound card PineTab

There's no upstream UCM for PineTab, yet. So I guess we can safely
rename it. Pine64-Arch is using this name.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
index 6056c965baf6..b7ab7d28914e 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
@@ -475,6 +475,7 @@ hdmi_out_con: endpoint {
 
 &sound {
 	status = "okay";
+	simple-audio-card,name = "PineTab";
 	simple-audio-card,aux-devs = <&codec_analog>, <&speaker_amp>;
 	simple-audio-card,widgets = "Microphone", "Internal Microphone Left",
 				    "Microphone", "Internal Microphone Right",
-- 
2.35.3

